credentials-list-directive.js

```diff
@@ -29,6 +31,9 @@ function factory($location, brAlertService, brCredentialService, config) {
     model.state = {
       credentials: {loading: true}
     };
+    model.showHeadline = scope.showHeadline === false ? false : true;
+    model.showPublicIndicator =
+      scope.showPublicIndicator === false ? false : true;
```

Comment:
It may be better/more extensible to do an array of display elements defined by enumeration:

```js
model.showElements = ['headline', 'publicIndicator'];
```

```html
<br-credentials-list br-show-elements="['headline', 'publicIndicator']" ... >
```

But that may also be overkill here. We can leave it as is, your call @mattcollier.